bin/build-all

```diff
@@ -13,6 +13,15 @@ ARMVAR=github.com/exercism/cli/cmd.BuildARM
 # handle alternate binary name for pre-releases
 BINNAME=${NAME:-exercism}
 
+get_shell_completions() {
+        SHELLCOMP_BASEURL=http://cli.exercism.io/shell/exercism_completion
```

Comment:
Let's keep the scripts in this repository instead of in the static HTML repo (as you suggested). That way this will be a simple file inclusion rather than a network call.